docs/examples/advanced/processes.ipynb

```diff
@@ -447,7 +447,7 @@
    "outputs": [],
    "source": [
     "class SimpleOscillator(nengo.Process):\n",
-    "    def make_step(self, shape_in, shape_out, dt, rng):\n",
+    "    def make_step(self, shape_in, shape_out, dt, rng, state=None):\n",
```

Comment:
Should the notebook explain the `state` parameter (since this is a notebook about how to make custom processes)?

Reply:
Done, thanks for pointing this out :ok_hand: